flake8_quotes/__init__.py

```diff
@@ -10,33 +10,57 @@ class QuoteChecker(object):
     version = __version__
 
     QUOTES = {
-        # When user wants only single quotes
+        # Allow single quotes
+        # Fail on double quotes
         '\'': {
-            'good_single': '\'',
-            'good_multiline': '\'\'\'',
-            'bad_single': '"',
-            'bad_multiline': '"""',
+            'allow': '\'',
+            'fail': '"',
```

Comment:
Can we move back to `good/bad`? It will cause less PR noise and they were adjectives describing the strings (which makes sense in conditionals), not verbs describing actions to perform against strings (makes less sense in conditionals unless they are methods).